docs/en/Use-Cases.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: global
+title: Use Cases
+group: Home
+---
+
+* Table of Contents
+{:toc}
+
+
+Here is a description of a few common use cases of Alluxio.
+
+## A Distributed Caching Layer for Data on Object Storage
+
+In many organizations, an increasingly popular architecture is to leverage the
+object storage as the single source of data for bigdata analytics jobs like
+Hadoop, Spark or Presto. Object storage can be public object storage services
+(e.g., AWS S3, Azure blob storage, Google Cloud Storage, Aliyun OSS, Tencent COS
+and etc) or the on-premise object stores (e.g., Ceph, Swift and etc).
+
+Though object stores are often more cost-effective, scalable, available and
+easier to maintain, they are not a "real" filesystem for a few reasons even
+they may provide a filesystem-like interface:
+
+- Lack of filesystem-level data caching so multiple jobs accessing the
+ same data set can benefit from caching the frequently accessed common
+ data.
+- Lack of node-level data locality on computation as data is always read
+ remotely and never node-local to the compute tasks.
+- Operations to list objects can be very slow especially when there is a massive
+ amount of objects in the bucket.
+- Operations to rename objects can be very slow and get weaker
+ consistency guarantees.
+
+In such architectures, deploying Alluxio on compute side and configured with
+data persisted at these object stores can greatly benefit the
+applications. Alluxio can not only cache the hot data local closer to the compute and
+possbiliy shared by different applications, but also manage the metadata
+separately to avoid certain inefficient metadata operations in object stores.
+
+See example use cases from
+[BazzarVoice](https://www.slideshare.net/ThaiBui7/hybrid-collaborative-tiered-storage-with-alluxio), [Myntra](http://alluxio-com-site-prod.s3.amazonaws.com/resource/media/myntra-case-study-accelerating-analytics-in-the-cloud-for-mobile-e-commerce).
+
+## ETL Elimination for Satellite Compute Clusters
+
+For reasons including performance, security or resource isolation, many
+organizations create satellite computation clusters (independent from their main
+Hadoop cluster) using dedicated resource for mission-critical
+applications. These satellite clusters often need to access data from the main
+Hadoop cluster, leading to the need to either read data remotely on job
+execution or run ETL pipelines to preload data into satellite clusters prior to
+jobs execution.
+
+Alluxio can accelerate the remote data read from main Hadoop cluster without
+adding extra ETL steps ahead. Deployed on the compute nodes on the satellite
+cluster and configured to connect to the main Hadoop cluster, an Alluxio service
+can serve like a local data proxy layer that provides the same namespace of the
+main HDFS data to the local compute jobs. In addition, Alluxio will
+transparently cache frequently accessed data to improve the locality and reduce
+network traffic of these jobs.
```

Comment:
Can add that Alluxio reduces load on the main cluster.

Reply:
added